.github/actions/poetry_setup/action.yml

```diff
@@ -0,0 +1,64 @@
+# An action for setting up poetry install with caching.
```

Comment:
is actions/poetry_setup/action.yml the convention? (vs just actions/poetry_setup.yml)

Reply:
Yeah my understanding is that this is how it's set up